.env

```diff
@@ -0,0 +1,48 @@
+# ==================
+# ==== Open edX ====
+# ==================
+
+# -- Django --
+SERVICE_VARIANT=lms
+DJANGO_SETTINGS_MODULE=lms.envs.fun.docker_run
+
+# -- Database --
+MYSQL_ROOT_PASSWORD=
+MYSQL_ALLOW_EMPTY_PASSWORD=yes
+MYSQL_DATABASE=edxapp
+MYSQL_USER=edxapp_user
+MYSQL_PASSWORD=password
+
+# -- Email --
+EMAIL_BACKEND=django.core.mail.backends.smtp.EmailBackend
+EMAIL_HOST=mailcatcher
+EMAIL_PORT=1025
+
+# -- Python --
+PYTHONUNBUFFERED=1
+
+# -- Queues --
+CELERY_ALWAYS_EAGER=1
+
+# -- Features --
+FEATURES={"AUTOMATIC_AUTH_FOR_TESTING": true, "RESTRICT_AUTOMATIC_AUTH": false}
```

Comment:
Maybe add one line to explain what this unlocks?

Reply:
Done.